### DISCUSSION TITLE
Help With Netbox and Blade Center H configuration

### DISCUSSION THREAD

**Discussion by amorpablo2024** [netbox-community/netbox#15313] [Help Wanted!]
Hello. I need to add in NETbox into a Blade Center H, 6 phisical servers inside with all the data. Im new on this and I dont understand very well the process. I edit the fields in the Blade Center H but only can change name but no add IP adresses or other importat data from the servers that are allocated inside.

Thanks in advance for the help

**Comment by candlerb** [comment]
The way to model a blade server is as a "parent" device for the chassis, which has "device bay" components for the slots, and then the blades themselves are additional devices which are "child" devices, each and associated with a specific device bay on the parent. Whether the device is a "parent" or "child" is set in its Device Type.

To add an IP address to a child device (or indeed any device), you need to create at least one interface on it, and then add the IP address to that interface. This interface can be a "virtual" one if you don't have any physical interface. Once you've done this, you can set the "primary IP address" of the device to this IP address - or in fact any IP address on any interface in the device.

As regarding "other important data" for the servers, what sort of data were you thinking of? It may be that Netbox has built-in features for what you're trying to do, but if not, you can define Custom Fields.

**Reply by amorpablo2024** [reply]
thanks for your time and the answer.. I will try to add the child  device as you mention!!
